src/datacatalogtordf/dataset.py

```diff
@@ -116,17 +116,17 @@ def frequency(self: Dataset, frequency: str) -> None:
         self._frequency = URI(frequency)
 
     @property
-    def spatial_coverage(self: Dataset) -> Union[Location, str]:
-        """Get/set for spatial_coverage."""
-        return self._spatial_coverage
+    def spatial(self: Dataset) -> List[Union[Location, str]]:
+        """Get/set for spatial."""
+        return self._spatial
 
-    @spatial_coverage.setter
-    def spatial_coverage(self: Dataset, spatial_coverage: Union[Location, str]) -> None:
-        self._spatial_coverage = spatial_coverage
+    @spatial.setter
+    def spatial(self: Dataset, spatial: List[Union[Location, str]]) -> None:
+        self._spatial = spatial
 
     @property
     def spatial_resolution(self: Dataset) -> Decimal:
-        """Get/set for spatial_resolution."""
+        """Get/set for spatial_resolution:."""
```

Comment:
Skal vel ikke være `:` her?
```suggestion
        """Get/set for spatial_resolution."""
```